rdrf/rdrf/views/landing_view.py

```diff
@@ -1,9 +1,11 @@
+from csp.decorators import csp_update
 from django.shortcuts import render
 from django.views.generic.base import View
 from rdrf.models.definition.models import Registry
 
 
 class LandingView(View):
+    @csp_update(DEFAULT_SRC=('https://fonts.googleapis.com', 'https://fonts.gstatic.com'))
```

Comment:
Shouldn't this be ``FONT_SRC`` instead of ``DEFAULT_SRC``?

Reply:
Same reason as here: https://github.com/eresearchqut/trrf/pull/225#discussion_r411036289

There is no inheritance, so we'd have to set a FONT_SRC in the settings

Reply:
Do we load fonts form anywhere else than those places though?
If not we don't need the ``FONT_SRC``, right? Or you can just set it to ``'self'``.